packages/variant-theme/src/ThemeGenerator.ts

```diff
@@ -0,0 +1,426 @@
+    colorSubtleBackgroundLightAlphaPressed: theme.colorSubtleBackgroundPressed
+  };
+
+  if (!isInverted) {
+    //Status colors need to change
+    newThemeProps.colorStatusWarningBackground1 = theme.colorStatusWarningForeground3;
+    newThemeProps.colorStatusWarningForeground3 = theme.colorStatusWarningBackground1;
+    newThemeProps.colorStatusDangerForeground1 = theme.colorStatusDangerBackground1;
+    newThemeProps.colorStatusDangerBackground1 = theme.colorStatusDangerForeground1;
+    newThemeProps.colorStatusSuccessForeground1 = theme.colorStatusSuccessBackground1;
+    newThemeProps.colorStatusSuccessBackground1 = theme.colorStatusSuccessForeground1;
+  }
+
+  newThemeProps.colorPaletteRedForeground1 = resetByContrast(
+    theme.colorPaletteRedForeground1,
+    brand[80]
+  );
+  newThemeProps.colorPaletteDarkOrangeForeground1 = resetByContrast(
+    theme.colorPaletteDarkOrangeForeground1,
+    brand[80]
+  );
+  newThemeProps.colorPaletteRedForeground3 = resetByContrast(
+    theme.colorPaletteRedForeground3,
+    brand[80]
+  );
+  newThemeProps.colorPaletteGreenForeground1 = resetByContrast(
+    theme.colorPaletteGreenForeground1,
+    brand[80]
+  );
+  newThemeProps.colorPaletteRedBorder2 = resetByContrast(
+    theme.colorPaletteRedBorder2,
+    brand[80]
+  );
+
+  Object.assign(theme, newThemeProps);
+}
+
+function isTokenNameContains(token: string, names: string[]): boolean {
+  return names.some((name) => token.indexOf(name) !== -1);
+}
+
+function shift(color: string, darken: boolean, degree: number) {
+  if (!greyReverse[color]) {
+    return color;
+  }
+  const index: number = Number(greyReverse[color]);
+  return darken
+    ? grey[Math.max(index - degree, 0) as Greys]
+    : grey[Math.min(index + degree, 100) as Greys];
+}
+
+function getNumberFromToken(token: string): number | undefined {
+  const nums = token.match(/\d+/);
+  return nums ? parseInt(nums[0]) : undefined;
+}
+
+function rgbaToHex(color: IColor, isInverted: boolean): string {
+  if (!color.a) {
+    return `#${color.r.toString(16)}${color.g.toString(16)}${color.b.toString(16)}`;
+  }
+  const r: number = Math.round(color.r * color.a * 0.01 + (!isInverted ? 255 * (1 - color.a * 0.01) : 0));
+  const g: number = Math.round(color.g * color.a * 0.01 + (!isInverted ? 255 * (1 - color.a * 0.01) : 0));
+  const b: number = Math.round(color.b * color.a * 0.01 + (!isInverted ? 255 * (1 - color.a * 0.01) : 0));
+  return `#${r.toString(16)}${g.toString(16)}${b.toString(16)}`;
+}
+
+function resetByContrast(
+  color: string | undefined,
+  pageBackground: string
+): string | undefined {
+  if (!color || !pageBackground) {
+    return color;
+  }
+  const colorContrastWithRed: number = contrast(
+    hex_to_sRGB(pageBackground),
+    hex_to_sRGB(color)
+  );
+  if (colorContrastWithRed < standardContrastRatio) {
+    return white;
+  }
+  return color;
+}
+
+export const calculateSoftBackground = (brandVariants, isInverted: boolean) => {
```

Comment:
export only things that are meant to be exported

```suggestion
const calculateSoftBackground = (brandVariants, isInverted: boolean) => {
```

Reply:
fixed